### PR TITLE
Cc/upsell/thanks switching v1

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
New screen for the thank you page after the upgrading screen where the user has switched. This is the first version of the screen as final designs are still to be signed off so wording and design may change. And actual monetary amounts need to be passed in.

## How to test
Storybook sceneario added or head to upgrade-support/thank-you

## Images
image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+						<span>
+							Your new support will start today. It can take up to
+							an hour for your support to be activated
+						</span>
+					</li>
+				</ul>
+			</section>
+			<section>
+				<div css={signInCss}>
+					<SwitchSignInImage />
```
Comment (andrewHEguardian) [inline client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx:80]: I wonder if this banner would be worth making a component at this point - is it exactly the same as on the switch page?

File: client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	signInContentContainerCss,
+	signInCss,
+	signInHeadingCss,
+	signInParaCss,
+} from '../../shared/SignIn';
+import {
+	headingCss,
+	stackedButtonLayoutCss,
+	whatHappensNextCss,
+} from '../cancel/cancellationSaves/SaveStyles';
```
Comment (andrewHEguardian) [inline client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx:22]: I think either we need to duplicate these styles or make them common somewhere else - feels odd to reference files from the switch and save journeys in this folder

File: client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx (hunk 1 above)
Comment (charleycampbell) [reply client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx:80]: I have made a SignIn shared file and was starting to refactor

File: client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx (hunk 2 above)
Comment (charleycampbell) [reply client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx:22]: Yeah agreed. It's whether to start touching files which aren't part of this change or not

Comment (andrewHEguardian) [reply client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx:22]: In general I'd say it's fine, if it's just moving things around

**Review by andrewHEguardian — APPROVED** [review]
Looks good 👍  let's refactor more as we go, on to the next screen 🚀

**Pull request merged**